worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/ClipboardTransformBaker.java

```diff
@@ -36,12 +37,10 @@
 /**
  * Helper class to 'bake' a transform into a clipboard.
  *
- * <p>This class needs a better name and may need to be made more generic.</p>
- *
  * @see Clipboard
  * @see Transform
  */
-class FlattenedClipboardTransform {
+public class ClipboardTransformBaker {
```

Comment:
Move to `internal.util`, add `Clipboard#transform(Transform);Clipboard` which delegates to `bakeTransform`.